creating_packages/package_repo.rst

```diff
@@ -172,8 +172,8 @@ be valid too:
 
 .. tip::
 
-    While you are in the same computer (the same Conan cache), even when you have exported the recipe and
-    Conan has captured the absolute url and commit, Conan will store the local folder where your source code lives.
-    If you build your package locally, it will use the local repository (in the local folder) instead of the remote URL,
-    even if the local directory contains uncommitted changes.
-    This allows you to speed up the development of your packages when cloning from a local repository.
+    When doing a :command:`conan create` of a recipe using ``scm``, Conan will store a local path to the local source repository. Every time
```

Comment:
Maybe rephrase a little bit: _"Conan will store in a file the path to the local source repository"_.